src/Compilers/CSharp/Test/Semantic/Semantics/TargetTypedDefaultTests.cs

```diff
@@ -325,6 +354,10 @@ static void Main()
             Assert.Null(model.GetTypeInfo(def).Type);
             Assert.Equal("System.Int32", model.GetTypeInfo(def).ConvertedType.ToTestDisplayString());
             Assert.Null(model.GetSymbolInfo(def).Symbol);
+            Assert.Equal(0, model.GetConstantValue(def).Value);
```

Comment:
It doesn't look like they agree.

---
In reply to: [102340395](https://github.com/dotnet/roslyn/pull/17205#discussion_r102340395) [](ancestors = 102340395)